### DISCUSSION TITLE
Call to a member function beginTransaction() on null

### DISCUSSION THREAD

**Discussion by vrvoicu** [jenssegers/laravel-mongodb#2113] [Q&A] [answered]
- Laravel-mongodb Version: 3.6.4
- PHP Version: 7.4
- Database Driver & Version: 3.2

### Description:
When running jobs i get "Call to a member function beginTransaction() on null"
Tried to extend the class but laravel hooks dont work
I suggest adding a workaround by overriding the methods getPdo(), beginTransaction(), commit(), rollBack() to empty function in Connection class

### Steps to reproduce
1. Register a job
2. Call php artisan queue:work

### Expected behaviour
When it comes to mongo it shoud run the job

### Actual behaviour
The getPdo() method returns null and beginTransaction() throws the error specified

<details><summary><b>Logs</b>:</summary>
Insert log.txt here (if necessary)
</details>

**Comment by Smolevich** [comment, marked as answer]
@vrvoicu what are settings in `config/queue.php` ?
